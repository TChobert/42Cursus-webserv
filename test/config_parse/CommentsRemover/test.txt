[A SIMPLE TEST WITH COMMENTS]

var=hello                                    #comment
var2 = hello #bis repetita
var3=#

#comment

abcd

#comment2

[ROUTE]

var=1234###########

##################################

line=line

123456#

#