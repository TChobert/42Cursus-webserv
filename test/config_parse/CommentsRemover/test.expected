[A SIMPLE TEST WITH COMMENTS]

var=hello                                    
var2 = hello 
var3=

abcd



[ROUTE]

var=1234

a

line=line

123456


